deployment/mcad-operator/deploy/operator.yaml

```diff
@@ -19,6 +19,18 @@ spec:
           # Replace this with the built image name
           image: darroyo/mcad-operator:v0.1.8
           imagePullPolicy: Always
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
```

Comment:
Is it really served by the MCAD controller? I can only find reference to the `/healthz` endpoint.

Reply:
you are correct, it wasn't being served at all.  an extremely rushed PR on my part I'm afraid. 

I've followed similar logic to what was already present for `healthz`